README.md

```diff
@@ -1 +1,179 @@
-# School-Library-OOP
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+
+<!-- MAIN HEADING -->
+
+  <h3><b>SCHOOL LIBRARY</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [� Table of Contents](#-table-of-contents)
+- [📖SCHOOL LIBRARY ](#school-library-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Prerequisites](#prerequisites)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- INTRO -->
+
+# 📖SCHOOL LIBRARY <a name="about-project"></a>
+
+> A tool to record books in the library and who borrows them.
+## 🛠 Built With <a name="built-with"></a>
+
+1. Ruby
+<!-- 2. CSS
+3. JavaScript
+4. Webpack -->
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://Ruby.org/">Ruby</a></li>
+    <!-- <li><a href="https://www.w3.org/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li> -->
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Ruby codes used to decode Morse Code
+> - List all Books
+> - Create a Person
+> - Create a Rental
+> - List all Rentals for a given Person id
+  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy of the project, use this link:
+
+```sh
+cd school library
+git@github.com:Kingno3l/School-Library-OOP.git
+```
+
+<!-- SETUP -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+To setup this project:
+
+```sh
+Download latest version at RubyInstaller for windows on local machine
+
+Download wsl -install -d Ubuntu for virtual experience
+```
+
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- INSTALL -->
+
+### Install
+
+Install this project with RubyInstaller on windows on local machine.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+irb
+```
+
+
+<!-- ### Run tests
+
+To test the project, execute the following command:
+
+```sh
+npm run test
+``` -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]() -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **King Immanuel**
+
+- GitHub: [@Kingno3l](https://github.com/Kingno3l)
+- Twitter: [@Kingno3l_](https://twitter.com/Kingno3lA)
```

Comment:
- [ ] When i tried verifying your X account it came up saying `this account does not exist`, please kindky provide the appropriate link to your X account :+1: :sparkles: